app/main.py

```diff
@@ -2,11 +2,11 @@ def calculate_team_total_rating(list_of_players: list) -> int:
     return sum([player.get_rating() for player in list_of_players])
 
 
-def elves_concert(list_of_elfs: list) -> None:
-    for i in range(len(list_of_elfs)):
-        list_of_elfs[i].play_elf_song()
+def elves_concert(list_of_elves: list) -> None:
+    for elv in list_of_elves:
```

Comment:
Now you should change **elv** to **elf**. :)

Reply:
:)

Reply:
sorry for that) i'm very careless